supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc

```diff
@@ -69,6 +69,17 @@
 
 *See also*: xref:management-cli[management CLI]
 
+[discrete]
+[[natively-compiled-code]]
+==== image:images/yes.png[yes] natively compiled code (noun)
+*Description*: _Natively compiled code_ is code written in a programming language that compiles to machine code with a resulting binary executable file. Such code can be run standalone.
```

Comment:
```suggestion
*Description*: _Natively compiled code_ is code written in a programming language that compiles to machine code with a resulting runnable binary file. Such code can be run standalone.
```

Reply:
As per the discussion with SME leaving the original version.